packages/ui/src/lib/dropdown/Dropdown.spec.ts

```diff
@@ -25,56 +25,57 @@ import Dropdown from './Dropdown.svelte';
 import DropdownTest from './DropdownTest.svelte';
 
 test('a button is created', async () => {
-  render(Dropdown);
+  render(Dropdown, { ariaLabel: 'Dropdown' });
 
-  const input = screen.getByRole('button');
+  const input = screen.getByRole('button', { name: 'Dropdown' });
   expect(input).toBeInTheDocument();
 });
 
 test('initial value is visible', async () => {
   render(Dropdown, {
     value: 'a value',
+    ariaLabel: 'Dropdown',
```

Comment:
@deboer-tim wondering why the ariaLabel is not using the name as default aria-label but use ''

https://github.com/containers/podman-desktop/blob/main/packages/ui/src/lib/dropdown/Dropdown.svelte#L21